pkgs/by-name/ro/rofi-unwrapped/package.nix

```
@@ -91,22 +94,29 @@ stdenv.mkDerivation rec {
     xcbutilxrm
   ];
 
-  mesonFlags =
-    lib.optionals x11Support [ "-Dimdkit=true" ]
-    ++ lib.optionals (!waylandSupport) [ "-Dwayland=disabled" ]
-    ++ lib.optionals (!x11Support) [ "-Dxcb=disabled" ];
+  mesonFlags = [
+    (lib.mesonBool "imdkit" x11Support)
+    (lib.mesonEnable "wayland" waylandSupport)
+    (lib.mesonEnable "xcb" x11Support)
+  ];
 
   doCheck = false;
 
-  meta = with lib; {
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+
+  meta = {
     description = "Window switcher, run dialog and dmenu replacement";
     homepage = "https://github.com/davatorium/rofi";
-    license = licenses.mit;
-    maintainers = with maintainers; [
+    changelog = "https://github.com/davatorium/rofi/releases/tag/${finalAttrs.version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [
       bew
       SchweGELBin
     ];
-    platforms = with platforms; linux;
+    platforms = with lib.platforms; linux;
```

Since we have only one, why not do this?
```suggestion
    platforms = lib.platforms.linux;
```